src/ui/Backdrop/Backdrop.ts

```diff
@@ -56,13 +61,24 @@ export class Backdrop extends Component {
      *
      * The default value is `false`.
      */
-    overlayGradient: ComponentOptions.buildBooleanOption({ defaultValue: false, depend: 'overlayColor' })
+    overlayGradient: ComponentOptions.buildBooleanOption({ defaultValue: false, depend: 'overlayColor' }),
+    /**
+     * If specified, the component will redirect the page to this URL when clicked.
+     */
+    clickUrl: ComponentOptions.buildStringOption(),
```

Comment:
And thinking of it, the Backdrop is a bit weird : we don't want another half implemented ResultLink component that kind of work correctly. 

Poke me when you read this @Dominionized and we'll talk live about this :)